This patch is based on the upstream commit described below, modified to
remove fuzz.

commit 45d5347a0d2187c046a546a477d2a53111cd7713
Author: Yuki Okushi <yuki.okushi@huawei.com>
Date:   Sat Sep 4 07:01:42 2021 +0900

    Rename `const_generics` feature gate to `adt_const_params`

Index: rust-packed-simd-0.3.3/src/lib.rs
===================================================================
--- rust-packed-simd-0.3.3.orig/src/lib.rs
+++ rust-packed-simd-0.3.3/src/lib.rs
@@ -200,7 +200,7 @@
 //!   preserving, etc.
 
 #![feature(
-    const_generics,
+    adt_const_params,
     repr_simd,
     platform_intrinsics,
     stdsimd,
